### PR TITLE
Update dind docker hub readme

### PR DESCRIPTION
**Pull request by alexcb**
This additionally moves the docker hub readme update scripts to a separate workflow file.

### CODE REVIEW

**Comment by alexcb** [comment]
Note that I'm currently working on a follow up PR that will include the docker version in the tags. That follow up PR will update this readme.

File: .github/workflows/update-docker-hub-readmes.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: Docker CI Ubuntu
+
+on:
+  push:
+    branches: [ main ]
```
Comment (nelsam) [inline .github/workflows/update-docker-hub-readmes.yml:5]: Should we include a `paths` option so we only run this job if the `docs` folder changes? `paths: [ 'docs/**' ]` would probably do it.

It's not a big deal, but with the GHA problems we seem to have, being more selective about when we run jobs might be useful. :)

**Review by nelsam — APPROVED** [review]
It all looks good. The `paths` selector in the workflow is not really important.

File: .github/workflows/update-docker-hub-readmes.yml (hunk 1 above)
Comment (alexcb) [reply .github/workflows/update-docker-hub-readmes.yml:5]: ah good call, i'll add that back in.

**Pull request merged**